### DISCUSSION TITLE
Is possible to override the module path?

### DISCUSSION THREAD

**Discussion by aurexav** [tokio-rs/tracing#2274] [Q&A]
Is possible to override the module path?

This is defined in util. But I don't want the module path to be always util.
```
impl Drop for ExecutionTimer {
	fn drop(&mut self) {
		tracing::debug!(
			target = self.target,
			"`{}` takes `{}` secs",
			self.meta,
			self.start.elapsed().as_secs_f64()
		);
	}
}
```

**Comment by davidbarsky** [comment]
If you create the metadata yourself, yes, but I'm curious as to why you want to change that module path. It's the first i've heard it be requested.

**Reply by aurexav** [reply]
Because I define this `ExecutionTimer` in a util library. And use it everywhere.
I don't want it to show this log comes from that util library. I want a more precise path.

**Reply by aurexav** [reply]
This is how I do this. https://github.com/hack-ink/subalfred/blob/main/src/subalfred/util.rs#L10-L12
I put it in a closure. But in this way, I need to write more code. I have to make a macro.
